SampleOfferWithoutReward.sol

```diff
@@ -160,8 +158,7 @@ contract SampleOfferWithoutReward {
 
     // "fire the contractor"
     function returnRemainingEther() onlyClient {
-        if (originalClient.DAOrewardAccount().call.value(this.balance)())
-            isContractValid = false;
+        originalClient.DAOrewardAccount().call.value(this.balance)();
```

Comment:
You need to wrap send functions in if statments because of the call stack attack http://martin.swende.se/blog/Devcon1-and-contract-security.html